### ISSUE TITLE
Crash after netplay is set to LAN

### ISSUE THREAD

**Issue by testman42** [EKA2L1/EKA2L1#455]
**Describe the bug**
EKA2L1 AppImage starts fine if `btnet-discovery-mode` in `config.yml` is set to `0`, but crashes if it is set to ˙2 ̇.

**To Reproduce**
 
Run EKA2L1
Go to Bluetooth -> Netplay -> Configure
Set mode to Local LAN (I'm quite sure that the L in LAN stands for "Local")
Click "Save" button
Close EKA2L1
Run it again.

if I use FIsh shell, the output in terminal is this:

```fish
Unhandled SIGSEGV at rip 0x0000564918f54874
fish: Job 1, './eka2l1-qt-x64.App…' terminated by signal SIGSEGV (Address boundary error)
```
However, if I run it with BASH, I get this:
```bash
Unhandled SIGSEGV at rip 0x000055f475ec3874
Segmentation fault (core dumped)
```

**Expected behavior**
EKA2L1 window should appear again.

**Your PC or smartphone specs**
OS: Manjaro 22.0.0 Sikaris
Kernel: x86_64 Linux 6.1.1-1-MANJARO
CPU: Intel Core i5-7200U @ 4x 3.1GHz [57.0°C]
GPU: AMD Radeon R7 M340 (iceland, LLVM 14.0.6, DRM 3.49, 6.1.1-1-MANJARO)

**Your emulator configuration**
G 04.10 Game Developer SW Variant (RH-29 - S60v1)

**Log**
> Attach your log file (EKA2L1.log), which is in same directory as your executable file
there is no log file next to the appimage
the log file in ~/.local/share/EKA2L1 is empty

**Comment by pent0** [comment]
Should recheck this now that netplay seems more OK

**Comment by testman42** [comment]
Nope, still getting the same issue with the latest [CI Build release](https://github.com/EKA2L1/EKA2L1/releases/tag/continous) (commit [c55f897](https://github.com/EKA2L1/EKA2L1/commit/c55f89740a1d42f0f7da319b25c7ae36b2b81703)).
Using Manjaro 22.1.0.

**Comment by testman42** [comment]
Issue persists with the latest CI Build release
